src/Nosto.php

```diff
@@ -83,6 +84,11 @@ public static function getServerUrl()
         return self::getEnvVariable('NOSTO_SERVER_URL', self::DEFAULT_NOSTO_SERVER_URL);
     }
 
+    public static function getEmailWidgetBaseUrl()
+    {
+        return self::getEnvVariable('NOSTO_EMAIL_WIDGET_BASE_URL', self::DEFAULT_NOSTO_EMAIL_WIDGET_BASE_URL);
```

Comment:
Add `NOSTO_EMAIL_WIDGET_BASE_URL ` to the `.env` file

Reply:
Fixed